Documentation/CommandReference/Index.rst

```diff
@@ -479,6 +479,18 @@ Arguments
 ``value``
   Value for system configuration
 
+Options
```

Comment:
The builds fail, because this is a manual addition to the reference, which is otherwise automatically generated. See https://github.com/TYPO3-Console/TYPO3-Console/blob/master/.github/CONTRIBUTING.md for hints how to set up for contributing and how to generate the reference automatically